oap-server/server-storage-plugin/storage-iotdb-plugin/pom.xml

```diff
@@ -44,6 +44,16 @@
         <dependency>
             <groupId>org.apache.iotdb</groupId>
             <artifactId>iotdb-session</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.jpountz.lz4</groupId>
+                    <artifactId>lz4</artifactId>
+                </exclusion>
```

Comment:
@jixuan1989 We manually replaced this with `1.6.0` to avoid 2 versions existing together in the final binary.